### PR TITLE
Add user model

### PR DESCRIPTION
**Pull request by adilrookout**
add user model

### CODE REVIEW

**Comment by adiludmer** [comment]
Hey Adi,

We see you'v added a new DB table to the project.

Is this table going to contain some PIIs ?

**Comment by adilrookout** [comment]
No

File: src/user_model.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from sqlalchemy import Column, Integer, String
+
+class User(Base):
```
Comment (adiludmer) [inline src/user_model.py:3]: Hey Adi,

We see you'v added a new DB table to the project.

Is this table going to contain some PIIs ?

Comment (adilrookout) [reply src/user_model.py:3]: No

**Pull request closed without merging**